### PR TITLE
CVE-2013-6627 and CVE-2015-1221

### PR DESCRIPTION
**Pull request by hollyhastings**

### CODE REVIEW

File: cves/CVE-2013-6627.yml [hunk 1]
```diff
@@ -4,21 +4,23 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: |
```
Comment (scioffi) [inline cves/CVE-2013-6627.yml]: the CWE value should just be the integer ID.
```suggestion
CWE: 119
```

File: cves/CVE-2015-1221.yml [hunk 2]
```diff
@@ -4,21 +4,21 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: CWE-416 Use After Free
```
Comment (scioffi) [inline cves/CVE-2015-1221.yml]: The CWE value should just be the CWE ID number.
```suggestion
CWE: 416
```

File: cves/CVE-2013-6627.yml [hunk 3]
```diff
@@ -32,39 +32,48 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  An attack can access information beyond the length of a buffer storing HTTP
```
Comment (KyleRohde) [inline cves/CVE-2013-6627.yml]: Split up into multiple sentences to help with understanding. Presents a lot of information in just one sentence

File: cves/CVE-2013-6627.yml [hunk 4]
```diff
@@ -77,9 +86,12 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
-  fix: 
+  answer: |
+    From looking at the fix code. It is obvious that the orginal code was
```
Comment (KyleRohde) [inline cves/CVE-2013-6627.yml]: Redundancy with 'original' and 'originally'

File: cves/CVE-2013-6627.yml [hunk 5]
```diff
@@ -94,21 +106,28 @@ discovered:
...
-    leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer: 
-  date: 
-  automated: 
-  google: 
-  contest: 
+    leave the entries blank except for "answer". Write down where you looked in
+    "answer".
+  answer: |
+    It is unclear how this vulnerability was initially discovered. However, the
```
Comment (KyleRohde) [inline cves/CVE-2013-6627.yml]: Split up second sentence or add commas: "...server, which demonstrates the vulnerability, and..."

File: cves/CVE-2013-6627.yml [hunk 6]
```diff
@@ -32,39 +32,48 @@ description_instructions: |
...
+    note:
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes: 2
```
Comment (KyleRohde) [inline cves/CVE-2013-6627.yml]: +3

File: cves/CVE-2013-6627.yml [hunk 7]
```diff
@@ -198,4 +231,19 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: |
+    The vulnerability appears to mainly stem from design mistakes. This is
+    evident because the code shows that the developers did not knowing that 1xx
```
Comment (KyleRohde) [inline cves/CVE-2013-6627.yml]: 'knowing' -> 'know'

File: cves/CVE-2013-6627.yml [hunk 8]
```diff
@@ -198,4 +231,19 @@ mistakes:
...
+    HTTP response status codes are different from the other classes of status
+    codes and because the developers made a lot of assumptions without
+    considering if the code written actually fits their assumptions.
+
+    Also, the vulnerability shows miscommunication issues regarding the
+    assumptions made (which influence which fix option was used). There also
+    seems to be little communication between people on the team which I believe
```
Comment (KyleRohde) [inline cves/CVE-2013-6627.yml]: Split up this sentence. Stating two different ideas with no punctuation.

File: cves/CVE-2015-1221.yml [hunk 9]
```diff
@@ -32,7 +32,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  The vulnerability found in Google's Blink Engine allows attackers to remotely
```
Comment (KyleRohde) [inline cves/CVE-2015-1221.yml]: Did the attack end in something more specific than just impacting service?

File: cves/CVE-2015-1221.yml [hunk 10]
```diff
@@ -41,31 +44,33 @@ bounty:
...
+    note:
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes:
```
Comment (KyleRohde) [inline cves/CVE-2015-1221.yml]: +0

File: cves/CVE-2015-1221.yml (hunk 9 above)
Comment (hollyhastings) [reply cves/CVE-2015-1221.yml]: They weren't really sure of the other impacts. I just added clarification about a possible denial of service and "unspecified other impact" (the wording actually used in the vulnerability description).

File: cves/CVE-2013-6627.yml [hunk 11]
```diff
@@ -132,12 +156,16 @@ major_events:
...
-  - name: 
-    date: 
-  - name: 
-    date: 
+    - name: |
+        A developer "touched" the code by moving it around. They did not know
+        that there was anything wrong with the code. He defended himself in the
+        issue thread because he believed he was being blamed for writing the
+        defective code
+      date:
```
Comment (moa1662) [inline cves/CVE-2013-6627.yml]: Can you find the date for this? If not, provide a link to this evidence.

File: cves/CVE-2015-1221.yml [hunk 12]
```diff
@@ -41,31 +45,33 @@ bounty:
...
+    note:
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes: 0
```
Comment (moa1662) [inline cves/CVE-2015-1221.yml]: +5

File: cves/CVE-2013-6627.yml [hunk 13]
```diff
@@ -32,39 +32,48 @@ description_instructions: |
...
+    note:
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes: 5
```
Comment (moa1662) [inline cves/CVE-2013-6627.yml]: +5

**Pull request merged**